pkg/metrics/metrics.go

```diff
@@ -884,6 +888,7 @@ func CreateConfiguration(metricsEnabled []string) (Configuration, []prometheus.C
 			c.ProxyReceivedEnabled = true
 
 		case Namespace + "_proxy_upstream_reply_seconds":
+			// TODO: Deprecate?
```

Comment:
Calling attention to this. How should we deprecate the old one, if we think that's the best thing to do? 

cc @michi-covalent

Reply:
for metrics we usually just delete them and mention it in the upgrade guide.

might make sense to have a general discussion with @cilium/metrics team regarding summary vs histogram though and set a guideline for when to use which one.

Reply:
agree yeah. For what it's worth, I think it would help to have a concrete proposed text to debate regarding summary/histogram because I have had conversations with different people on different PRs, and I've heard varied opinions on these. Might be easier to propose some text and then use that as the center for the discussion.

Also happy to include this as a topic during an upcoming weekly community meeting. Feel free to add it to open session items [here](https://docs.google.com/document/d/1Y_4chDk4rznD6UgXPlPvn3Dc7l-ZutGajUv1eF0VDwQ/edit#).

Reply:
Put it up for discussion for next week.